client/my-sites/checkout/src/components/item-variation-picker/styles.tsx

```diff
@@ -83,12 +82,13 @@ export const OptionList = styled.ul`
 	}
 `;
 
-export const Discount = styled.span`
+export const Discount = styled.span< { shouldUseCheckoutV2: boolean } >`
 	color: ${ ( props ) => props.theme.colors.discount };
 	margin-right: 8px;
-	${ hasCheckoutVersion( '2' )
-		? `align-items: left; font-size: 12px;`
-		: `align-items: center; font-size: 100%;` }
+	${ ( props ) =>
+		props.shouldUseCheckoutV2
+			? `align-items: left; font-size: 14px;`
```

Comment:
I didn't notice anything wrong in the UI but I see that this was changed from 12px to 14px. Was that intentional?

Reply:
I think this was intentional, but since it isn't covered in the Figma I was just trying to bring things in line with the new standard 14px font-size. I'm fine with either font-size though.

Reply:
I ended up removing these styles as the 12px font looked better in some instances.